src/sage/combinat/gelfand_tsetlin_patterns.py

```diff
@@ -1003,7 +1003,7 @@ def _row_iter(self, upper_row):
                 pos -= 1
                 continue
             # If it would create an invalid entry, backstep
-            if ( pos > 0 and (row[pos] >= row[pos-1] \
+            if ( pos > 0 and (row[pos] >= row[pos-1]
                     or (self._strict and row[pos] == row[pos-1]-1)) ) \
                     or row[pos] >= upper_row[pos] \
                     or (self._k is not None and row[pos] >= self._k):
```

Comment:
`\` in lines 1007, 1008 are necessary?

Reply:
yes, they are

Reply:
I see. Aligning `or` in the line 1007 together with `or` in lines 1008, 1009 caused a confusion. Wouldn't it be better to join 1006 and 1007 together? Perhaps this again breaks PEP8. But I think this weighs more.

Reply:
let's keep that for another ticket, please

Reply:
Okay. No problem to keep it for now.